src/lib/server-utilities.js

```diff
@@ -0,0 +1,9 @@
+import fs from 'fs';
+import crypto from 'crypto';
+
+export function md5Hash(file) {
+  const hashInstance = crypto.createHash('md5');
+  // readFileSync in the syncronous version of readFile
+  const fileContents = fs.readFileSync(file, 'utf8');
+  return hashInstance.update(fileContents).digest('hex');
+}
```

Comment:
For the first point, I think the dashed filenames work well. [This StackOverflow answer](https://stackoverflow.com/a/7273448) makes a good argument for it as well (and against using capital letters in standard filenames).

As for your second point, I prefer to use `.jsx` instead of `.js` just to differentiate the files and make the difference a bit more clear for anyone reading the code. Our `.jsx` files (which are all React components) follow quite a different structure than a standard `.js` file would. Dan Abramov's argument makes sense, but I think it makes more sense for experienced developers who can understand where JSX is being used and where it's just vanilla JavaScript.